react/src/components/lablupTalkativotUI/LLMPlaygroundPage.tsx

```diff
@@ -20,17 +20,32 @@ const LLMPlaygroundPage: React.FC<LLMPlaygroundPageProps> = ({ ...props }) => {
 
   const [isSynchronous, setSynchronous] = useState(false);
 
-  const { endpoint_list } = useLazyLoadQuery<LLMPlaygroundPageQuery>(
+  const urlParams = new URLSearchParams(window.location.search);
+  const endpointId = urlParams.get('endpointId');
+  const modelId = urlParams.get('modelId');
+  const isEmptyEndpointId = !endpointId;
```

Comment:
Instead of access `window.location`, it's better to use `useQueryParam` for controlling params.